etcdctl/ctlv3/command/make_mirror_command.go

```diff
@@ -45,7 +48,8 @@ func NewMakeMirrorCommand() *cobra.Command {
 	}
 
 	c.Flags().StringVar(&mmprefix, "prefix", "", "Key-value prefix to mirror")
-	// TODO: add dest-prefix to mirror a prefix to a different prefix in the destination cluster?
+	c.Flags().StringVar(&mmdestprefix, "dest-prefix", "", "destination prefix to mirror a prefix to a different prefix in the destination cluster")
+	c.Flags().BoolVar(&mmremdestprefix, "rem-dest-prefix", false, "remove the key-value prefix in the destination cluster")
```

Comment:
rem-dest-prefix is not readable. So I suggested to change `rem-dest-prefix` to `no-dest-prefix`. And yes, I also want to change the description to `mirror key-values to the root of the destination cluster`.

I will be more clear next time. Thanks!